kuma/payments/jinja2/payments/terms.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ page_title(_('MDN Payment terms')) }}{% endblock %}
+
+{% block site_meta %}
+    {{ super() }}
+    {% set seo_description = _('MDN Payment terms') %}
+    {% set social_title = _('MDN Payment terms') %}
+    <meta property="og:description" content="{{ seo_description }}">
+    <meta name="description" content="{{ seo_description }}">
+    <meta name="twitter:description" content="{{ seo_description }}">
+    <meta name="twitter:title" content="{{ social_title }}">
+    <meta property="og:title" content="{{ social_title }}">
+{% endblock %}
+
+
+{% block content %}
+    <div>
+        <h2>Payment terms</h2>
+        <p>
+            Mozilla (that’s us) operates MDN Web Docs.
+        </p>
+        {% set url="https://www.mozilla.org/en-US/about/legal/terms/mozilla/" %}
+        <p>
+            These Payment Terms cover your payment to MDN. <a href="{{ url }}">The Websites & Communications Terms of Use</a>
```

Comment:
I can't believe a web company used an ampersand & instead of spelling out "and" 🤦‍♂️